setup.py

```diff
@@ -67,39 +69,9 @@ def run(self):
 #  Dependencies
 #
 
-INSTALL_REQUIRES = [
-
-    # NuCypher
-    'umbral==0.1.1a3',
-    'constant-sorrow',
-    'bytestringSplitter',
-    'hendrix>=3.1.0',
-
-    # Third Party (General)
-    'cryptography>=2.3',
-    'pysha3',
-    'requests',
-    'sqlalchemy',
-    'apistar<0.6.0',
-    'tzlocal<2.0.0',
-    'maya',
-
-    # Third Party (Ethereum)
-    'eth-utils==1.2.2',
-    'eth-keys',
-    'eth-tester>=0.1.0b33',
-    'py-evm>=0.2.0a33',
-    'py-solc',
-    'web3',
-
-    # Third Party (Configuration + CLI)
-    'appdirs',
-    'click>=7.0',
-    'colorama',
-    'sentry-sdk==0.5.2',
-    'boto3',
+with open(os.path.join(BASE_DIR, "requirements.txt")) as f:
```

Comment:
Is this safe?

Feels kinda odd, are any other projects doing this?

Reply:
Yes - many.  What is the concern?

Reply:
Just the idea that a modified requirements.txt can cause it to install the depends. Just curious.